### ISSUE TITLE
[Feature Request] Patch Merge

### ISSUE THREAD

**Issue by Midaychi** [WojciechKrysiak/SCModManager#32]
Simply, an option to only export the files that are changed in a merge, so you are essentially creating a patch instead of copying all the files from each respective mod.

IE: 
If you had mod a and mod b, right now it will create mod a+b with the changes/overrides you select in addition. 

With patch merging, you'd merge mod a+b and the resulting merge would only contain the conflict winner files and wouldn't import anything else from mod a or b.

**Comment by Gorehuchi** [comment]
I agree, this would drastically increase the usefulness of the manager. At least to me anyway, as I prefer fixing one problem at a time.

**Comment by WojciechKrysiak** [comment]
There's work underway on making the manager multi-platform, and the feature you request is included there - the newest releases can be found at https://ci.appveyor.com/project/WojciechKrysiak/scmodmanager/build/artifacts and the code is at https://github.com/WojciechKrysiak/SCModManager/tree/feature/PortToAvalonia
